nomic/atlas.py

```diff
@@ -76,9 +78,7 @@ def map_embeddings(
         description = description
 
     if data is None:
-        data = [{
-            ATLAS_DEFAULT_ID_FIELD: b64int(i)
-        } for i in range(len(embeddings))]
+        data = [{ATLAS_DEFAULT_ID_FIELD: b64int(i)} for i in range(len(embeddings))]
```

Comment:
it would be great if we either enforce users to upload with an id field or find a way to pass that to the user so they can map back from `datum_id` to their original data. I've had to rebuild maps a few too many times since I forgot to add an id column and couldn't join with the tags I had added to the data points

Reply:
yeah, they just shouldn't be using map_embeddings or map_text.
its kinda a hard problem to solve.

Reply:
I will add information that these are set in upload order.

Reply:
I think we could write a warning to stderr on map_embeddings and map_text.

Reply:
I will add a warning and also documentation.

Reply:
What should people be using instead of map_embeddings or map_text then?

Reply:
p = AtlasProject()

p.add_text()

p.create_index()